### ISSUE TITLE
RadioButton does not work

### ISSUE THREAD

**Issue by UlfR** [xotahal/react-native-material-ui#25]
Seems `onCheck={this.onPress}` missed in render

**Comment by xotahal** [comment]
Yeah, RadioButton needs refactor :/ There are bad props, bad default props, and useless onPress function :/ But it should work anyway. Because it just renders Checkbox with another icons. I'll try it ;)

**Comment by UlfR** [comment]
May be it because i`ve used them in RadioButtonGroup copy, but without adding onCheck prop to <RadioButton> call, buttong ignore changing
